src/SDKs/HDInsight/Management.HDInsight/Customizations/Converters/DefaultVmSizes.cs

```diff
@@ -45,6 +46,7 @@ public static class WorkerNode
             {
                 {"spark", "Standard_D12"},
                 {"rserver", "Standard_D4_v2"},
+                {"mlservices", "Standard_D4_v2"},
```

Comment:
These are additive changes to the SDK. Please update the AssemblyVersion to `3.0.2-preview`

Reply:
Good catch. Fixed.